### PR TITLE
website: updated linux installation docs for 0.8

### PR DESCRIPTION
**Pull request by deekay2310**
Signed-off-by: Dev Kumar <devskumar99@gmail.com>

### What does this PR do?

Updated installation documentation for Podman Desktop on Linux

### Screenshot/screencast of this PR

Screen Shot 2022-10-11 at 1 27 58 PM
Screen Shot 2022-10-11 at 1 27 53 PM

### CODE REVIEW

File: website/docs/Installation/linux-install.md [hunk 1]
```diff
@@ -14,31 +14,39 @@ Now that you have Podman machine up and running, you can check out the [Download
 
 ## Installing Podman Desktop
```
Comment (slemeur) [inline website/docs/Installation/linux-install.md:12]: Should be nice to have a small sentence in the intro about the dependencies.
I think it's also where we should callout that podman is rootless.

**Review by slemeur — CHANGES_REQUESTED** [review]
Minor changes.
Please keep the structure consistent between all install guides. It should have a "Next Steps" section as on the Mac one.

File: website/docs/Installation/linux-install.md [hunk 2]
```diff
@@ -14,31 +14,39 @@ Now that you have Podman machine up and running, you can check out the [Download
...
+With FlatHub, you can install Podman Desktop with just a single command.
 
-Open the Terminal and go to Downloads directory. 
+> Make sure to follow the [setup guide](https://flatpak.org/setup/) before installing
 
 ```sh
-cd Downloads
+flatpak install --user flathub io.podman_desktop.PodmanDesktop
 ```
+After installation, to open Podman Desktop:
```
Comment (cdrage) [inline website/docs/Installation/linux-install.md]:
```suggestion
After installation, open Podman Desktop:
```

File: website/docs/Installation/linux-install.md [hunk 3]
```diff
@@ -14,31 +14,39 @@ Now that you have Podman machine up and running, you can check out the [Download
...
 
-> In order to install & start the application, you need to know the Flatpak identifier for the application. In our case, that is `io.podman_desktop.PodmanDesktop` starting 0.0.6 and `com.github.containers.desktop` for previous versions
+```sh
+flatpak run io.podman_desktop.PodmanDesktop
+```
 
-Run the following command to install the flatpak application,
+### 2. Using [flatpak](https://flatpak.org/setup/) application from [Downloads](/downloads/linux)
+
+Before you follow along, make sure flatpak is set up in your distribution. Click [here](https://flatpak.org/setup/) to learn more about setting up flatpak.
```
Comment (cdrage) [inline website/docs/Installation/linux-install.md]:
```suggestion
**Prerequisites:** [Install Flatpak](https://flatpak.org/setup/) before proceeding with the following commands.
```

May be better and since it's bolded / shows to the user better?

File: website/docs/Installation/linux-install.md [hunk 4]
```diff
@@ -14,31 +14,39 @@ Now that you have Podman machine up and running, you can check out the [Download
...
+
+Open the Terminal and go to Downloads directory. 
 
 ```sh
-flatpak install <name_of_the_flatpak_file> 
+cd Downloads
 ```
 
-For example:
+> In order to install & start the application, you need to know the Flatpak identifier for the application, that is `io.podman_desktop.PodmanDesktop`.
```
Comment (cdrage) [inline website/docs/Installation/linux-install.md]:
```suggestion
> The Flatpak identifier is `io.podman_desktop.PodmanDesktop` and will be used for installation purposes.
```

Sounds a bit more formal / straight to the point?

File: website/docs/Installation/linux-install.md [hunk 5]
```diff
@@ -14,31 +14,39 @@ Now that you have Podman machine up and running, you can check out the [Download
...
+
+Run the following command to install the flatpak application,
```
Comment (cdrage) [inline website/docs/Installation/linux-install.md]:
```suggestion
Run the following command to install the Flatpak application,
```

IMO all references to Flatpak should be capitalized.

File: website/docs/Installation/linux-install.md [hunk 6]
```diff
@@ -14,31 +14,39 @@ Now that you have Podman machine up and running, you can check out the [Download
...
 ```
+After installation, to open Podman Desktop:
 
-> In order to install & start the application, you need to know the Flatpak identifier for the application. In our case, that is `io.podman_desktop.PodmanDesktop` starting 0.0.6 and `com.github.containers.desktop` for previous versions
+```sh
+flatpak run io.podman_desktop.PodmanDesktop
+```
 
-Run the following command to install the flatpak application,
+### 2. Using [flatpak](https://flatpak.org/setup/) application from [Downloads](/downloads/linux)
```
Comment (cdrage) [inline website/docs/Installation/linux-install.md]:
```suggestion
### 2. Using [flatpak](https://flatpak.org/setup/) on Linux
```

It's weird that `[Downloads]` is linked to `/downloads/linux` in this title?

**Review by cdrage — CHANGES_REQUESTED** [review]

File: website/docs/Installation/linux-install.md (hunk 1 above)
Comment (deekay2310) [reply website/docs/Installation/linux-install.md:12]: has been added in the latest commit

File: website/docs/Installation/linux-install.md (hunk 3 above)
Comment (deekay2310) [reply website/docs/Installation/linux-install.md]: I've used the admonition from docusaurus for this in the latest commit, it's a good way to provide a prerequisite.

File: website/docs/Installation/linux-install.md (hunk 6 above)
Comment (deekay2310) [reply website/docs/Installation/linux-install.md]: it takes you to the downloads page from where you can obtain the flatpak or the zip file

**Review by slemeur — APPROVED** [review]
LGTM

**Comment by benoitf** [comment]
@deekay2310 FYI, when merging, you should probably squash the commits (select the other option in the dropdown) if they're about the same subject. We don't need intermediate commits to land into the main branch.

**Pull request merged**